offload/CMakeLists.txt

```diff
@@ -10,6 +10,13 @@
 #
 ##===----------------------------------------------------------------------===##
 
+cmake_minimum_required(VERSION 3.20.0)
+
+if (OPENMP_STANDALONE_BUILD OR "${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
+  set(OPENMP_STANDALONE_BUILD TRUE)
+  project(omptarget C CXX ASM)
+endif()
```

Comment:
Why "OpenMP?" Shouldn't it of `offload`? This is definitely required for standalone compilation however.

Reply:
Because the libomptarget folder was just renamed offload, all of the checks were not renamed. So we stuck with OPENMP_STANDALONE_BUILD.

Reply:
I meant the `project` name as well.

What is this pattern
```cmake
if (OPENMP_STANDALONE_BUILD)
  set(OPENMP_STANDALONE_BUILD TRUE)
endif()
```
Seems wrong.

Reply:
Sure, the project could probably change. If the offload/CMakeLists.txt is invoked directly without setting `OPENMP_STANDALONE_BUILD` in the cmake configuration then `OPENMP_STANDALONE_BUILD` will need to be set here so the rest of the checks will work.